.packit.yml

```diff
@@ -0,0 +1,32 @@
+# Docs: https://packit.dev/docs/
+---
+
+specfile_path: strace.spec
+upstream_package_name: strace
+downstream_package_name: strace
+
+# list of packages needed for bootstrap, make dist-gzip and rpm build
+srpm_build_deps:
+  - autoconf
+  - automake
+  - make
+  - libtool
+
+actions:
+  # generate spec file from input file
+  post-upstream-clone:
+    - ./bootstrap
+    - ./configure --disable-dependency-tracking --enable-Werror --enable-maintainer-mode
+  create-archive:
+    - make dist-gzip
+    - bash -c "ls ./strace-*.tar.gz"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
```

Comment:
Is it possible to trigger this also in case of a push?

Reply:
Yes it is, I'll update the configuration